### DISCUSSION TITLE
How to create context/dropdown menu?

### DISCUSSION THREAD

**Discussion by qarmin** [slint-ui/slint#4588] [Q&A]
I wanted to implement this two types of floating menus, but I cannot find proper widgets, that allows to do this

![Screenshot from 2024-02-09 10-29-20](https://github.com/slint-ui/slint/assets/41945903/8821e354-1453-4b67-8bdc-6e4ee4fad981)

![Screenshot from 2024-02-09 10-33-09](https://github.com/slint-ui/slint/assets/41945903/d6752bac-812a-46e3-b2d6-aed01e27f8ea)

**Comment by ghost** [comment]
Hi,
I have implemented a `ContextMenu` this way:
 
* implementation: https://codeberg.org/flovansl/co_sl/src/branch/main/widgets/coop/ui/widgets/context_menu.slint
* usage: https://codeberg.org/flovansl/co_sl/src/commit/dfddbbf06e1fb133741988e976542fd0483a0d67/apps/coop_local/ui/views/files_view.slint#L68
